### ISSUE TITLE
TTS button in problem instructions

### ISSUE THREAD

**Issue by benetechMatt** [benetech/MathShare#996]

**Comment by benetechMatt** [comment]
![mockup showing the TTS speaker button to the left of the problem prompt at the top of the problem screen.  The button is white to match the text which is on an orange background.](https://user-images.githubusercontent.com/49413678/75797404-ba1a6600-5d3a-11ea-827f-5082d0436a9a.png)

**Comment by alexrcabral** [comment]
Is this needed? The prompt is the first step in any given equation and cannot be edited or deleted.

**Comment by alexrcabral** [comment]
this is complete as of v1.4.0 release
